venvs/make.py

```diff
@@ -10,12 +10,31 @@
 
 from functools import partial
 
+import filesystems
 from filesystems import Path
 from packaging.requirements import Requirement
 import click
+import tomlkit
 
 from venvs import __version__
-from venvs.common import _FILESYSTEM, _LINK_DIR, _ROOT
+from venvs.common import _FILESYSTEM, _LINK_DIR, _ROOT, dump_config, load_config
+
+
+def add_virtualenv_config(filesystem, locator, installs, links):
+    try:
+        contents = load_config(filesystem=filesystem, locator=locator)
+    except filesystems.exceptions.FileNotFound:
+        contents = tomlkit.table()
+        contents.add('virtualenv', {})
```

Comment:
`setdefault` would be what you want here (and below) though it's hard to tell because this thing appears to inherit from `dict` :( https://github.com/sdispater/tomlkit/blob/master/tomlkit/container.py#L26